Plugin breaks if git config log.showSignature is 'true'

### Describe the bug
If the user's gitconfig has this entry:
```
[log]
        showsignature = true
```
then if the commit is GPG-signed, then the plugin breaks with 
```
Caused by: java.lang.NumberFormatException: For input string: "gpg: Signature made Thu 18 Mar 2021 10:58:04 PM UTC
gpg:                using RSA key A889DE190F2C51C7F006C5F27C2A95403085B982
gpg: Good signature from "... user" [ultimate]
1616108284"
    at java.lang.NumberFormatException.forInputString (NumberFormatException.java:65)
    at java.lang.Long.parseLong (Long.java:692)
    at java.lang.Long.parseLong (Long.java:817)
    at pl.project13.core.NativeGitProvider.getCommitTime (NativeGitProvider.java:255)
    at pl.project13.core.GitDataProvider.maybePut (GitDataProvider.java:259)
    at pl.project13.core.GitDataProvider.loadGitData (GitDataProvider.java:161)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitDataWithNativeGit (GitCommitIdMojo.java:691)
    at pl.project13.maven.git.GitCommitIdMojo.loadGitData (GitCommitIdMojo.java:668)
    at pl.project13.maven.git.GitCommitIdMojo.execute (GitCommitIdMojo.java:516)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:566)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
````

The problem is actually simple: `NativeGitProvider::getCommitTime` has a bug, it executes `git log -1 --pretty=format:%ct` instead of `log -1 --pretty=format:%ct --no-show-signature`. I'll submit a PR.

### Steps to Reproduce
- Please include the **full** configuration of the plugin

```xml
 <plugin>
                <groupId>pl.project13.maven</groupId>
                <artifactId>git-commit-id-plugin</artifactId>
                <version>${git-commit-id-plugin.version}</version>
                <inherited>false</inherited>
                <executions>
                    <execution>
                        <id>git-properties</id>
                        <phase>initialize</phase>
                        <goals>
                            <goal>revision</goal>
                        </goals>
                        <configuration>
                            <useNativeGit>true</useNativeGit>
                            <offline>true</offline
                            <dotGitDirectory>${basedir}/.git</dotGitDirectory>
                            <generateGitPropertiesFile>true</generateGitPropertiesFile>
                            <generateGitPropertiesFilename>${basedir}/buildinfo.properties</generateGitPropertiesFilename>
                            <skipPoms>false</skipPoms  
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

### Additional context
Not adding additional context as the problem is clear.

--- a/core/src/main/java/pl/project13/core/NativeGitProvider.java
+++ b/core/src/main/java/pl/project13/core/NativeGitProvider.java
@@ -230,41 +230,41 @@ public class NativeGitProvider extends GitDataProvider {
 
   @Override
   public String getCommitAuthorName() throws GitCommitIdExecutionException {
-    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%an " + evaluateOnCommit);
+    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%an --no-show-signature " + evaluateOnCommit);
   }
 
   @Override
   public String getCommitAuthorEmail() throws GitCommitIdExecutionException {
-    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%ae " + evaluateOnCommit);
+    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%ae --no-show-signature " + evaluateOnCommit);
   }
 
   @Override
   public String getCommitMessageFull() throws GitCommitIdExecutionException {
-    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%B " + evaluateOnCommit);
+    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%B --no-show-signature " + evaluateOnCommit);
   }
 
   @Override
   public String getCommitMessageShort() throws GitCommitIdExecutionException {
-    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%s " + evaluateOnCommit);
+    return runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%s --no-show-signature " + evaluateOnCommit);
   }
 
   @Override
   public String getCommitTime() throws GitCommitIdExecutionException {
-    String value =  runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%ct " + evaluateOnCommit);
+    String value =  runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%ct --no-show-signature " + evaluateOnCommit);
     SimpleDateFormat smf = getSimpleDateFormatWithTimeZone();
     return smf.format(Long.parseLong(value) * 1000L);
   }
 
   @Override
   public String getCommitAuthorTime() throws GitCommitIdExecutionException {
-    String value =  runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%at " + evaluateOnCommit);
+    String value =  runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%at --no-show-signature " + evaluateOnCommit);
     SimpleDateFormat smf = getSimpleDateFormatWithTimeZone();
     return smf.format(Long.parseLong(value) * 1000L);
   }
 
   @Override
   public String getCommitCommitterTime() throws GitCommitIdExecutionException {
-    String value =  runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%ct " + evaluateOnCommit);
+    String value =  runQuietGitCommand(canonical, nativeGitTimeoutInMs, "log -1 --pretty=format:%ct --no-show-signature " + evaluateOnCommit);
     SimpleDateFormat smf = getSimpleDateFormatWithTimeZone();
     return smf.format(Long.parseLong(value) * 1000L);
   }

